src/NServiceBus.AcceptanceTests/Recoverability/When_message_is_deferred_by_delayed_retries_using_dtc.cs

```diff
@@ -23,11 +24,10 @@ public class When_message_is_deferred_by_delayed_retries_using_dtc : NServiceBus
                         Id = c.Id
                     }))
                  )
-                .WithEndpoint<ErrorSpy>()
-                .Done(c => c.MessageMovedToErrorQueue)
+                .Done(c => c.FailedMessages.Any())
                 .Run();
 
-            Assert.Greater(context.NumberOfProcessingAttempts, 1, "Should retry at least once");
+            Assert.Greater(context.NumberOfProcessingAttempts, 3, "Should retry at least once");
```

Comment:
the text doesn't really match, but the test would have succeeded even when retries were disabled completely, as there is always 1 processing attempt + additional retry attempts. as delayed retries have been configured to 3, there should be a total of 4 invocations (>3)